src/picotorrent/application.cpp

```diff
@@ -102,11 +104,21 @@ bool Application::OnInit()
 
     auto cfg = std::make_shared<pt::Core::Configuration>(db);
 
+    // Load current locale
     pt::UI::Translator& translator = pt::UI::Translator::GetInstance();
     translator.LoadDatabase(env->GetCoreDbFilePath());
     translator.SetLocale(
         cfg->Get<std::string>("locale_name")
             .value_or(env->GetCurrentLocale()));
+    
+    // Load theme
+    pt::UI::Theming &theming = pt::UI::Theming::GetInstance();
+    theming.LoadThemes();
+    theming.SetCurrentTheme(cfg->Get<std::string>("theme_id").value_or("system")); // Sets default to 'System'.
+    if (theming.GetMSWDarkMode())

```

Comment:
If we go with a configuration based approach, this can be simplified to `cfg->IsDarkMode()`